orc8r/gateway/go/status/host_info.go

```diff
@@ -78,10 +78,7 @@ func init() {
 	}
 	disksInfo, _ = disk.Partitions(true)
 
-	w, err := routewrapper.NewRouteWrapper()
-	if err == nil {
-		hostRoutes, _ = w.Routes()
-	}
+	hostRoutes, _ = netlink.RouteList(nil, 0)
```

Comment:
Maybe you can use `netlink.RouteList(nil, netlink.FAMILY_V4)` here to avoid ipv6 addresses instead of filtering later.

Reply:
Turns out this is the right idea. I made the change.